apps/meeting/src/providers/AppStateProvider.tsx

```diff
@@ -33,11 +35,13 @@ export function useAppState(): AppStateValue {
 
 const query = new URLSearchParams(location.search);
 
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 export function AppStateProvider({ children }: Props) {
   const [meetingId, setMeeting] = useState(query.get('meetingId') || '');
   const [region, setRegion] = useState(query.get('region') || '');
   const [meetingMode, setMeetingMode] = useState(MeetingMode.Attendee);
   const [localUserName, setLocalName] = useState('');
+  const [isWebAudioEnabled, setIsWebAudioEnabled] = useState(false);
```

Comment:
Why false? Can we default this to true? I always join meetings with voice focus.

Reply:
@michhyun1  The main reason to disable web audio by default is to fix the audio fade issue in safari browser which was raised by intuit while using the react demo.